tsconfig.json

```diff
@@ -1,20 +1,20 @@
 {
 	"compilerOptions": {
-		"target": "es5",
-		"lib": ["dom", "dom.iterable", "esnext"],
+		"target": "ES2017",
```

Comment:
lowercase `es2017`, nu is het een mix van upper- en lowercase.

Reply:
Fixed